Renamed images, updated+shuffled blurbs.

kimjetwav opened the pull request:
# Renamed images, updated+shuffled blurbs.
I renamed all image files from their various schemes back to the
original scheme of ppl.f.last.jpg. I also normalised all variations on
JPG, JPEG, jpeg, and jpg, down to `jpg` for lexicographic purposes.

I shuffled people into and out of Alumni and current Members mostly on
Colin’s request, and updated about 4 or 5 blurbs and several pictures.

I removed the gigantic space-filling `stock.jpg` images for people
lacking headshots.

Added a few links to people’s twitters and things, and removed Nick’s
links on his request.

I updated the lab pic, the current one is `KimelLab2019.jpg`.

I stuck in a comment under Nick’s name and under Christin’s name
because as of now they both could do with at least one last follow up.

Pls feedback + test view to make sure it’s all right (it renders fine
for me, but I haven’t fully tested it on like the actual .io site
architecture).

Also, here is a list of members sorted by how the site index has them
sorted. Who are we missing and who is in the wrong column? Question
marks for people I wasn’t sure what to do with.

```
| Current    | Alumni    |
|------------+-----------|
| AristotleV |           |
| ColinH     |           |
| ErinD      |           |
| NickN      |           |
| LindsayO   |           |
| PeterZ     |           |
| HajerN     |           |
| IskaM-E    |           |
| MichaelJ   |           |
| JerryJ     |           |
| EmanN      |           |
| KevinW     |           |
| NedaR-R    |           |
| SalimM     |           |
| DawnS      |           |
| JustinN    |           |
| GraceJ     |           |
| JovankaS   |           |
| ThomasT    |           |
| ChristinS? | ChristinS?|
| AnaghaV?   | AnaghaV?  |
| SonjaS     |           |
| NavonaC    |           |
| JudyK?     | JudyK?    |
| DielleM    |           |
|------------+-----------|
|            | NatF      |
|            | GabiH     |
|            | JohnA     |
|            | HideakiT  |
| MathuM?    | MathuM?   |
|            | EllaW     |
|            | SophieM   |
|            | SineadR   |
|            | LaurenL   |
|            | DayonM    |
|            | LaagiY    |
|            | ErikaZ    |
| YuliyaN?   | YuliyaN?  |
|            | LauraS    |
|            | SabaS     |
|            | TomW      |
|            | DawsonO   |
|            | DanF      |
|            | JulieW    |
|            | MallarC   |
|            | TristramL |
|            | AnneW     |
| DavidR?    | DavidR?   |
|            | MikkoM    |
|            | TinaB     |
|            | RobertA   |
|            | MattP     |
|            | JonP      |
|            | MélissaL  |
|            | ArashN    |
|            | TinaR     |
|            | NikhilB   |
|            | AmyM      |
|            | VincentM  |
|            | JosephV   |
|            | ZoeM?     |
|            | LauraG?   |
```

kimjetwav committed:
Renamed images, updated+shuffled blurbs.

josephmje commented:
Thanks @kimjetwav for putting this all together! For some reason, Christin's blurb shows up in the render under Nick's picture. I think a lot of the middle section may be commented out? Also, Anagah can be moved to lab alumni. We're also missing a pic and blurb from Giulia. RE the Slaight peeps, should we keep Dielle and Judy? We don't have profiles for anyone else yet.

kimjetwav committed:
New update, +more pics & blurbs.

kimjetwav committed:
Improved and reworded my own blurb, & updooted my headshot. Ugh.

kimjetwav commented:
Hey, I fixed the issue with Christin’s blurb appearing below Nick’s headshot, it was just a comment in Nick’s section that I accidentally didn’t close properly and it ran all the way to a comment in Christin’s section. I also added some new blurbs and headshots from folks, and updated my own just because why not? Christin is still trying to produce a new blurb but she hasn’t got back to me, though I reminded her about it last night.

kimjetwav committed:
Shuffled Jo up higher.

kimjetwav commented:
We can merge more later if needed, but this seems fine for now. The people who never got back to me on the updated blurbs can wait for the next one.
